src/_posts/2018-01-09-beam-a-look-back.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title:  "Apache Beam: A Look Back at 2017"
+date:   2018-01-09 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - ianand
+  - jbonofre
+---
+
+On January 10, 2017, Apache Beam got [promoted]({{ site.baseurl }}/blog/2017/01/10/beam-graduates.html)
+as a Top-Level Apache Software Foundation project. It was an important milestone
+that validated the value of the project, legitimacy of its community, and
+heralded its growing adoption. In the past year, Apache Beam has been on a
+phenomenal growth trajectory, with significant growth in its community and
+feature set. Let us walk you through some of the notable achievements.
+
+<!--more-->
+
+## Use cases
+
+First, lets take a glimpse at how Beam was used in 2017. Apache Beam being a
+unified framework for batch and stream processing, enables a very wide spectrum
+of diverse use cases. Here are some use cases that exemplify the versatility of
+Beam.
+
+<img class="center-block"
```

Comment:
the jenkins verification job is failing due to image needing alt text:
image /images/blog/2017-look-back/timeline.png does not have an alt attribute (line 110)

I also think the image might need to be cropped to remove extra blank space around the graphic? from looking at staged version, there's a pretty big empty space below:
http://apache-beam-website-pull-requests.storage.googleapis.com/370/blog/2018/01/09/beam-a-look-back.html